app/src/main/java/com/codingblocks/cbonlineapp/PaddingLessBottomNavigation.kt

```diff
@@ -0,0 +1,23 @@
+package com.codingblocks.cbonlineapp
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+class PaddingLessBottomNavigation(context : Context, attributeSet: AttributeSet) : BottomNavigationView(context, attributeSet) {
```

Comment:
We don't this anymore i guess

Reply:
Ok

Reply:
will remove this in next commit and rebase it